ex3B/ex3.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Write a function that is given an array of numbers and a number and returns
+ * whether the number exists within the array.
+ *
+ * For example:
+ * containsValue([1,3,5,7,9], 8) === false;
+ * containsValue([1,3,5,7,9], 7) === true;
+ * containsValue([9,7,5,3,1], 7) === true;
+ */
+function containsValue(arrayList, searchValue) {
```

Comment:
In computer science: An array is a continuos piece of memory that contains some values. A list is some values in some order. As you can see "Array""List" doesn't necessarily reflect what you had in mind. How about array, arr, list, valueArr, valueList, or maybe just values.